### ISSUE TITLE
Integrate plugin to read index from remote

### ISSUE THREAD

**Issue by DmitryKey** [DmitryKey/luke#68]
Revisit and integrate:
https://github.com/md6nguyen/luke-remoteindex

**Comment by mocobeta** [comment]
Great integration!

**Comment by DmitryKey** [comment]
yes, if this works out, the problem of slow streaming UI via X server is addressed.

**Comment by mocobeta** [comment]
Lucene is basically designed for local file systems, so accessing to remote index is not a recommended way, I think. 
A web application looks more suitable for this purpose to me, so I created an issue for this: #122
